src/main/java/org/bytedeco/javacpp/Loader.java

```diff
@@ -1276,38 +1273,12 @@ public static String load(Class cls, Properties properties, boolean pathsFirst)
                         }
                     }
                 }
-                for (String executable : executables) {
-                    String[] split = executable.split("#");
-                    filename = prefix + split[0] + suffix;
-                    String filename2 = split.length > 1 ? prefix + split[1] + suffix : null;
-                    for (int i = extensions.length - 1; i >= -1; i--) {
-                        // iterate extensions in reverse to be consistent with the overriding of properties
-                        String extension = i >= 0 ? extensions[i] : "";
-                        String subdir = libraryPath.length() > 0 ? "/" + libraryPath : platform + (extension == null ? "" : extension);
-                        URL u = findResource(cls, subdir + "/" + filename);
-                        if (u != null) {
-                            if (filename2 != null) {
-                                u = new URL(u + "#" + filename2);
-                                if (!u.toString().contains("#")) {
-                                    Field f = URL.class.getDeclaredField("ref");
-                                    f.setAccessible(true);
-                                    f.set(u, filename2);
-                                }
-                            }
-                            File f = cacheResource(u);
-                            if (f != null) {
-                                f.setExecutable(true);
-                                executablePaths.add(f.getAbsolutePath());
-                            }
-                        }
-                    }
-                }
-            } catch (IOException | NoSuchFieldException | IllegalAccessException e) {
```

Comment:
This piece of code has been moved to a private static `listExecutables` method because both Loader.load and Loader.loadExecutable need access to this piece of code. Instead of returning a `List<String>` of all the executable paths it now returns a mapping between executable name and executable path.